stat/distmv/normal.go

```diff
@@ -341,7 +344,7 @@ func (n *Normal) ScoreInput(score, x []float64) []float64 {
 	copy(tmp, x)
 	floats.Sub(tmp, n.mu)
 
-	n.chol.SolveVecTo(mat.NewVecDense(len(score), score), mat.NewVecDense(len(tmp), tmp))
+	_ = n.chol.SolveVecTo(mat.NewVecDense(len(score), score), mat.NewVecDense(len(tmp), tmp))
```

Comment:
Are we comfortable with this? Does it deserved a comment?

Reply:
Maybe grab the err and panic if it's not okay? Right now this is fine because it cannot err if the factorization is okay. The check is at least defensive against changes.

Reply:
Done